draft-ietf-quic-transport.md

```diff
@@ -2575,7 +2576,7 @@ affected stream.
 
 Stream 1 is critical to the functioning of the entire connection.  If stream 1
 is closed with either a RST_STREAM or STREAM frame bearing the FIN flag, an
-endpoint MUST generate a connection error of type QUIC_CLOSED_CRITICAL_STREAM.
+endpoint MUST generate a connection error of type PROTOCOL_VIOLATION.
```

Comment:
Why remove this error? It's strictly more information, and is surely useful for debugging on the side that's likely messing up.